src/index.ts

```diff
@@ -15,13 +16,38 @@ nunjucks.configure('views', {
 
 app.get('/', (req: Request, res: Response) => {
 	console.log('Hello, client has sent get request');
-	res.render('index.njk');
+	res.render('index.njk', { electedRepresentativeName: '' });
```

Comment:
Probably will add a placeholder value here instead of empty string, just hadn't decided what to render (could use html placeholder attribute instead in the frontend?)